test/EntityFramework6.Npgsql.Tests/EntityFrameworkBasicTests.cs

```diff
@@ -772,5 +772,67 @@ public void TestTableValuedStoredFunctions()
                 Assert.AreEqual(1, list2[0].Something);
             }
         }
+
+        [Test]
+        public void Test_issue_60_and_62()
+        {
+            using (var context = new BloggingContext(ConnectionString))
+            {
+                context.Database.Log = Console.Out.WriteLine;
+
+                context.Blogs.Add( new Blog { Name = "Hello" });
+                context.SaveChanges();
+
+                string stringValue = "string_value";
+                var query = context.Blogs.Select(b => stringValue + "_postfix");
+                var blogTitle = query.First();
+                Assert.That(blogTitle, Is.EqualTo("string_value_postfix"));
+                Console.WriteLine(query.ToString());
+                StringAssert.AreEqualIgnoringCase("SELECT  COALESCE( @p__linq__0,E'')  || E'_postfix' AS \"C1\" FROM \"dbo\".\"Blogs\" AS \"Extent1\"", query.ToString());
```

Comment:
Readability: This line is too long and is hard to read for other developers, break it at sensible points into 2 or more strings concatenated with "+" so that the length of each line is as near in length to surrounding code. Also there is extra spaces around COALESCE statement. You don't have to add these spaces in the expression. The parsing system automatically handles that.